mod_reforged/hooks/config/rumors.nut

```diff
@@ -0,0 +1,68 @@
+// We add up 4 new variables for texts to be build with.
+// Three are wrong distances, direction and terrain for the purpose of creating more interesting rumors.
+// One is an indirect adjective for a legendary location if you don't wanna name it directly
+// This hook will add those vars for contracts and events aswell but they will not make use of those
+local buildTextFromTemplate = ::buildTextFromTemplate;
+::buildTextFromTemplate = function( _text, _vars )
```

Comment:
Since we only actually use this fake info in `tavern_building`, this should just be hooking `buildText` in there instead.

Reply:
I don't see how that would be possible.
I have no way accessing the `_vars` while hooking the `buildText` function. It's being assembled in that funtion and then passed directly to the `buildTextFromTemplate`